index.js

```diff
@@ -1,88 +1,123 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
+// let counter = 1;
+const counter = 2
```

Comment:
For this, you should use `let` instead instead of `const` because `let` allows us to reassign our variables